package.json

```diff
@@ -1,62 +1,132 @@
 {
   "name": "@testing-library/svelte",
-  "version": "2.0.0",
+  "version": "0.0.0-semantically-released",
   "description": "Simple and complete Svelte testing utilities that encourage good testing practices.",
   "main": "dist/index.js",
+  "types": "types/index.d.ts",
+  "license": "MIT",
+  "homepage": "https://github.com/testing-library/svelte-testing-library#readme",
   "repository": {
     "type": "git",
     "url": "https://github.com/testing-library/svelte-testing-library"
   },
+  "bugs": {
+    "url": "https://github.com/testing-library/svelte-testing-library/issues"
+  },
+  "engines": {
+    "node": ">= 8"
+  },
+  "keywords": [
+    "testing",
+    "svelte",
+    "ui",
+    "dom",
+    "jsdom",
+    "unit",
+    "integration",
+    "functional",
+    "end-to-end",
+    "e2e"
+  ],
+  "files": [
+    "dist",
+    "dont-cleanup-after-each.js",
+    "pure.js",
+    "types/index.d.ts"
+  ],
+  "scripts": {
+    "toc": "doctoc README.md",
+    "lint": "eslint src/**/*.js --fix",
+    "clean": "rimraf dist",
+    "build": "npm run clean && babel src --out-dir dist --ignore '**/__tests__/**'",
```

Comment:
You don't need a UMD or CJS bundle for this? All the other changes seem fine (I don't know Svelte) but keeping the build/test commands running through kcd-scripts seemed like a good idea given all the other ecosystem packages use it for bundling and other commands

Reply:
The main export is a CJS bundle via Babel. I don't know of a use case for a UMD bundle? In the event anyone needs it we can really easily add it using [microbundle](https://github.com/developit/microbundle). 

The build process is so simple that I think it's better to be clear to other contributors and maintainers what is going on when these scripts run. It also adds an extra level of flexibility to add anything project specific to the builds and understanding the consequences of doing so. The `preact-testing-library` has been good so far with the same build setup.

Of course that's just my opinion, if the testing-library uses the kcd-scripts for cross-project consistency then I can change these when I have time :)

Reply:
There are some use cases for UMD exports. Specifically people wanting to use these libraries in tools like puppeteer/testcafe. I think this is fine if you want to do it that way. Feel free to maintain this library as it suits you. `kcd-scripts` will always be there to help if you'd like :)